scrapper.py

```diff
@@ -1,6 +1,21 @@
 """
 Scrapper implementation
 """
+from datetime import datetime
+from pathlib import Path
+import random
+import re
+import shutil
+import time
+import json
```

Comment:
it is a better practice to sort your imports alphabetically

Reply:
Are you talking about the fact that pylint requires importing firstly standard packages, then external and then local? 
That is true, you should group them that way and import in this order.
However, within these blocks you are excepted to sort your imports alphabetically as this way your code is more readable